google/cloud/discoveryengine/v1/.repo-metadata.json

```diff
@@ -0,0 +1,14 @@
+{
+    "api_id": "discoveryengine.googleapis.com",
+    "api_shortname": "discoveryengine",
+    "client_documentation": "https://cloud.google.com/cpp/reference/discoveryengine/latest",
+    "distribution_name": "google-cloud-cpp",
+    "issue_tracker": "https://issuetracker.google.com/issues?q=componentid:911831%20status=open",
+    "language": "cpp",
+    "library_type": "GAPIC_AUTO",
+    "name_pretty": "Discovery Engine API",
+    "product_documentation": "https://cloud.google.com/generative-ai-app-builder/docs",
```

Comment:
FYI: it seems like the source data is wrong for this service:

https://github.com/googleapis/googleapis/blob/4221b78ecddf8edf168ff2817ed6c14dd844ae2c/google/cloud/discoveryengine/v1/discoveryengine_v1.yaml#L111-L112

The easiest thing is to open a bug internally and just leave it be as-is.  The next time we regenerate the library it will get fixed.

Reply:
Good catch.  I saw your CL, and will keep an eye on it.

Reply:
The conclusion on the CL was that these are the "right" links, so leaving it alone.